tests/consensus_tests/test_strict_mode.py

```diff
@@ -116,3 +116,84 @@ def test_strict_mode_upsert_local_shard(tmp_path: pathlib.Path):
 
     assert False, "Should have blocked upsert but didn't"
 
+
+def test_payload_strict_mode_upsert(tmp_path: pathlib.Path):
+    # TODO: Update this test when payload rocksDB storage has been replaced by mmap payload storage!
+    # Every required change for the mmap migration has been marked with a "TODO"
+    peer_urls, peer_dirs, bootstrap_url = start_cluster(tmp_path, 4)
+
+    strict_mode = {
+        "enabled": True,
+        "max_collection_payload_size_bytes": 8000,
+    }
+    create_collection(peer_urls[0], collection=COLLECTION_NAME, shard_number=4, replication_factor=N_REPLICAS, strict_mode=strict_mode)
+
+    wait_collection_exists_and_active_on_all_peers(collection_name=COLLECTION_NAME, peer_api_uris=peer_urls)
+
+    # Insert points into leader
+    for i in range(10):
+        upsert_random_points(peer_urls[0], 100, collection_name=COLLECTION_NAME, offset=i*100)
+        time.sleep(5)  # TODO: Remove
```

Comment:
Can we remove it now (and others)? :smiley:

Reply:
This is related to #5669 as well and can be removed when the payload storage has been migrated to mmap